ocpp/charge_point.py

```diff
@@ -53,7 +53,7 @@ def snake_to_camel_case(data):
     if isinstance(data, dict):
         camel_case_dict = {}
         for key, value in data.items():
-            key = key.replace("soc", "SoC")
+            key = key.replace("soc", "SOC").replace("_SOCket", "Socket")
```

Comment:
`snake_to_camel_case()` converts terms from idiomatic Python to the casing that the OCPP spec uses.
In the context of "State of Charge", OCPP uses the term `SoC`, not `SOC`.

See for example in [NotifyEVChargingNeedsRequest.json
](https://github.com/mobilityhouse/ocpp/blob/ae02ede445ea56375505d21bd498a88fce937862/ocpp/v201/schemas/NotifyEVChargingNeedsRequest.json#L127).

Make sure that `soc` is mapped to `SoC` in the context of "State of Charge". But that it is not converted in other contexts, for example in the context of `socket`.

Reply:
Ok, thanks for clear definition - I'll make the changes

Reply:
Satisfied in https://github.com/mobilityhouse/ocpp/pull/607/commits/1ad08b70a8d562d141559907f0e1560b8be0154e